components/checkbox/index.jsx

```diff
@@ -143,7 +148,31 @@ const propTypes = {
 	/**
 	 * Which UX pattern of checkbox? The default is `base` while other option is `toggle`. (**Note:** `toggle` variant does not support the `indeterminate` feature, because [SLDS does not support it](https://lightningdesignsystem.com/components/forms/#flavor-checkbox-toggle-checkbox-toggle).)
 	 */
-	variant: PropTypes.oneOf(['base', 'toggle', 'button-group']),
+	variant: PropTypes.oneOf(['base', 'toggle', 'button-group', 'visual-picker']),
+	/**
+	 * Determines whether visual picker is coverable when selected (only for visual picker variant)
+	 */
+	coverable: PropTypes.bool,
+	/**
+	 * Determines whether the visual picker should be vertical or horizontal (only for visual picker variant)
+	 */
+	vertical: PropTypes.bool,
+	/**
+	 * Allows icon to shown with checkbox (only for non-coverable visual picker variant)
+	 */
+	onRenderVisualPicker: PropTypes.node,
```

Comment:
`onRender` callbacks should be `function` instead of type node

Although, I guess that isn't very clear here and assumes that:
https://github.com/salesforce/design-system-react/blob/master/docs/codebase-overview.md#prefix-render-props-with-onrender